frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -4,15 +4,15 @@ import { appHost } from '../protractor.conf';
 import * as loginView from '../views/login.view';
 
 const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 4;
 const KUBEADMIN_IDP = 'kube:admin';
 const KUBEADMIN_USERNAME = 'kubeadmin';
 const { BRIDGE_KUBEADMIN_PASSWORD } = process.env;
 
 describe('Auth test', () => {
   beforeAll(async () => {
     await browser.get(appHost);
-    await browser.sleep(6000); // Wait long enough for the login redirect to complete
+    await browser.sleep(8000); // Wait long enough for the login redirect to complete
```

Comment:
We should determine whether the tests are failing because

1. It takes the login page too long to load
2. It takes too long to redirect back after login

If the problem is (1), changing the browser sleep here makes sense. If it's (2), we probably need to increase the timeout for this call: https://github.com/openshift/console/blob/master/frontend/integration-tests/views/login.view.ts#L29

Reply:
If we do change this sleep, I might give it more cushion like 10s.

Reply:
Added some debugging output, but this flake doesn't happen often:
```
Test Suites & Specs:

1) Auth test
----> getting https://console-openshift-console.apps.jephilli-4-8-04-20-0629.devcluster.openshift.com
----> sleeping 5 seconds
   2) Login test
----> in login
----> wait for visibility of username input
----> type username
----> type password
----> click submit
----> wait for userDropdown
----> after login
      ✔ logs in as kubeadmin user
```

The difficult part will be seeing this flake.

Reply:
@dtaylor113 We might have to merge some debugging log messages so we catch it in CI and don't have to keep retesting this PR.

Reply:
@dtaylor113 Let's increase the sleep at this spot to 10s and rollback the other changes in the PR. It sounds like the we're not waiting long enough for the select IDP page to load.

Reply:
@spadgett Done :-)